cmake/modules/hunter_add_package.cmake

```diff
@@ -9,6 +9,17 @@ include(hunter_internal_error)
 include(hunter_status_debug)
 include(hunter_test_string_not_empty)
 
+function(_hunter_find_project_dir projectname output)
```

Comment:
Should be in separate file with name `hunter_find_project_dir.cmake`. See "one function - one file" rule:
* http://0.readthedocs.org/en/latest/cmake.html#library-of-cmake-extra-modules